website/project/decorators.py

```diff
@@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+
+def dev_only(func):
```

Comment:
Currently the API only works for situations where the API server is launched, which means it can be merged into staging and production without fear, but the settings page won't have that luxury. We are going to need it to be running on staging2 and develop well before it goes into production, which means we need a way of limiting it on Production.

Since staging2 is in develop mode, that means that we could have the api running with this UI but not have it show in Production. The downside is that staging is also in develop mode, which will complicate things. So we may need another flag. It might be that we could generalize this decorator so that you pass in the appropriate value/flag rather than having it know about a specific flag. That would make it potentially more usable for different circumstances. 

All that being said, when the API launches in production, the decorator should no longer be used for this particular item.